client/packages/system/src/Patient/hooks/usePatientModalStore.ts

```diff
@@ -0,0 +1,31 @@
+import { zustand } from '@openmsupply-client/common';
```

Comment:
I would like to combine all these stores into a single one, but thought that I'd done too much already